src/test/groovy/org/shipkit/internal/gradle/VersioningPluginTest.groovy

```diff
@@ -56,8 +56,28 @@ class VersioningPluginTest extends PluginSpecification {
         versionInfo.notableVersions == [] as LinkedList
     }
 
-    def "adds version bumped changes to GitCommitTask if GitPlugin applied"() {
+    def "should add version bumped changes to GitCommitTask if GitPlugin applied"() {
         given:
+        project.file(VersioningPlugin.VERSION_FILE_NAME) << "version=0.9.0\npreviousVersion=0.8.114\n"
+
+        and:
+        project.plugins.apply(ReleaseConfigurationPlugin).configuration.gitHub.repository = "http://github.com"
+        project.plugins.apply(GitPlugin)
+
+        when:
+        project.plugins.apply(VersioningPlugin)
+
+        then:
+        GitCommitTask gitCommitTask = project.tasks.getByName(GitPlugin.GIT_COMMIT_TASK)
+        gitCommitTask.files.contains(project.file(VersioningPlugin.VERSION_FILE_NAME).absolutePath)
+        gitCommitTask.aggregatedCommitMessage.contains("0.9.0 release (previous 0.8.114)")
+    }
+
+    def "should skip previous version in release commit message if not available"() {
```

Comment:
Nitpicky (and since you know very well how to test stuff, the discussion might be good!)

I'd be tempted to just create unit test for 'formatVersionInformationInCommitMessage'  method to simplify the test and avoid retesting the coupling between versioning plugin and git plugin.